doc/sys/registry.delete.html

```diff
@@ -1,274 +1,262 @@
+      </a>
+      <a class="bd-navbar-icon navbar-item"
+         href="https://discord.gg/cvQV77nhfr"
+         target="_blank">
+        <span class="icon" style="color: #5865F2">
+          <i class="fab fa-lg fa-discord"></i>
+        </span>
+      </a>  
+    </div>
+  </nav>
+  
+  <section class="main-content columns is-centered is-fullheight" style="margin-top: 6%; margin-bottom: 4%">
+    <div class="column is-7">
+      <div class="card">
+        <a id="back" class="button is-white pl-5" href="index.html?methods"><span class="icon has-text-grey-lighter"><i class="fas fa-circle-left"></i></span></a>
+        <div class="card-content pt-0">
+          <div class="content">
+            <p class="has-text-weight-light is-size-3 mb-5" style="color: #004080; vertical-align: middle">sys.registry.delete(<span style="color:#EA6AC3">root</span>, <span style="color:#EA6AC3">key</span>, <span style="color:#ae81ff">[value]</span>)</p>
+            <p>Removes value(s) at the specified key in the Windows registry.</p>
+            <p class="is-size-4 mb-1 mt-6" style="color: #004080">Parameters</p>
+            <p class="has-text-weight-light is-size-5 mt-2 mb-0"><span style="color:#EA6AC3">root</span></p>
+            <p>Indicates the predefined entry point in the Windows registry, from which the key/value will be accessed. Must be one of the following string :              
+                <ul>
+                    <li><span style="color:#EA6AC3">"HKEY_CLASSES_ROOT"</span> : types (or classes) of documents and the properties associated with those types</li>
+                    <li><span style="color:#EA6AC3">"HKEY_CURRENT_CONFIG"</span> : information about the current hardware</li>
+                    <li><span style="color:#EA6AC3">"HKEY_CURRENT_USER"</span> : preferences of the current user</li>
+                    <li><span style="color:#EA6AC3">"HKEY_LOCAL_MACHINE"</span> : data about the computer, system memory, and installed hardware and software</li>
+                </ul></p>
+            <p class="has-text-weight-light is-size-5 mt-2 mb-0"><span style="color:#EA6AC3">key</span></p>
+            <p>Indicates the name of the key under which the value will be written.</p>                                 
+          <p class="has-text-weight-light is-size-5 mt-2 mb-0"><span style="color:#ae81ff">[value]</span></p>
+          <p>Indicates the specific value to be deleted, removes all values for the key if omitted </p>               
+          <p class="is-size-4 mb-1 mt-6" style="color: #004080">Return value</p>
+            This function returns <code>true</code> if the value has been removed from the registry, or otherwise <code>false</code> (See <a href="./error.html">sys.error</a> to get the error message).
+          
+            <p class="is-size-4 mt-6 mb-0" style="color: #004080">Example</p>
+            <div class="language-lua mr-6 ml-6"><code>
+local removed = sys.registry.delete("HKEY_CURRENT_USER", "Environment", "Path")
```

Comment:
This example is dangerous ! User may have problems with misconfigured PATH
You can replace it with : 

local removed = sys.registry.delete("HKEY_CURRENT_USER", "Environment", "value")